docker-compose.yml

```diff
@@ -20,8 +20,11 @@ services:
     container_name: mathesar_service
     environment:
       - MODE=${MODE-PRODUCTION}
+      - DJANGO_ALLOW_ASYNC_UNSAFE=true
+      - DISPLAY=${DISPLAY}
```

Comment:
DISPLAY cannot be passed from our `.env` file currently because pytest has a different environment configuration specified in `setup.cfg`

We need to allow usage of a separate env file for tests.